docs/topics/mta-assessment-changes-700.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+:_content-type: REFERENCE
+[id="mta-assessment-changes-700_{context}"]
+= Assessment module introduction
+
+The *Assessment* module in {ProductFullName} offers the following functionality for assessing and analyzing applications:
+
+.Assessment hub
+
+The *Assessment* hub integrates tightly with the *Application inventory*.
+
+.Enhanced questionnaire capabilities
+
+Previously, {ProductShortName} supported a single, built-in questionnaire. {ProductShortName} 7.0 not only includes that questionnaire, but also supports importing and exporting questionnaires. You can design custom questionnaires using YAML syntax, with a downloadable template provided. The syntax provides:
+
+* Conditional questions: The syntax supports skipping questions if a certain tag is present on the application or archetype.
+* Application auto-tagging according to answers: The syntax supports defining tags to be applied to applications if a certain answer has been provided.
+* Automated answers from tags in application archetypes
+
+For more details, see xref:mta-custom-questionnaire_user-interface-guide[Custom questionnaire].
+
+The default questionnaire remains in {ProductShortName} but can now be customized and saved. For more details, see xref:mta-default-questionnaire_user-interface-guide[Default questionnaire].
+
+.Multiple questionnaires
+
+Assessment supports multiple questionnaires, relevant to one or more applications.
+
+.Archetypes
+
+You can group applications with similar characteristics into archetypes. This allows you to assess multiple applications at once, rather having to assess each one separately. Each archetype has a shared taxonomy of tags, stakeholders, and stakeholder groups. All associated applications inherit assessment and review from the archetypes to which they belong. Automated answers from provided by tags in application archetypes.
```

Comment:
Might be worth mentioning how the archetype tag related field names on archetype creation align with this. - 
 - "Criteria tags" being tags that the archetype requires to include an application as a member. 
 - "Archetype tags" are tags that are applied to the group of applications that is known as an archetype. Will be applied to the archetype entity. 

Note: Criteria tags do not partially match an application. If for example application-a only has tag-a, but the archetype-a criteria tags include tag-a AND tag-b, application-a will not be a member of archetype-a. 


<img width="910" alt="Screenshot 2024-01-29 at 9 26 09 AM" src="https://github.com/windup/windup-documentation/assets/11218376/bf8446ef-18ac-4630-bba4-76562e36ea4a">

Reply:
Changed here

Note added to `mta-web-archetypes.adoc`